client/proxy.conf.json

```diff
@@ -1,4 +1,9 @@
 {
+  "/configure/*":{
```

Comment:
Can we just change the endpoint name to /_/configure where it is used on the client and where it is mapped in the server then we dont' need this block?